aafd/logwrapper.te

```diff
@@ -13,3 +13,10 @@ set_prop(logwrapper, vendor_video_codec_prop)
 set_prop(logwrapper, vendor_hwcomposer_prop)
 
 set_prop(logwrapper, vendor_usb_controller_prop)
+
+allow logwrapper p9fs:dir search;
+allow logwrapper p9fs:file r_file_perms;
+allow logwrapper p9fs:dir r_dir_perms;
+allow logwrapper p9fs:file { getattr open read };
```

Comment:
remove this line, duplicate with line 18

Reply:
updated